docs/en-US/component/tag.md

```diff
@@ -75,23 +75,23 @@ tag/checkable
 
 ## Attributes
 
-| Name                | Description                          | Type    | Accepted Values             | Default |
-| ------------------- | ------------------------------------ | ------- | --------------------------- | ------- |
-| type                | component type                       | string  | success/info/warning/danger | —       |
-| closable            | whether Tag can be removed           | boolean | —                           | false   |
-| disable-transitions | whether to disable animations        | boolean | —                           | false   |
-| hit                 | whether Tag has a highlighted border | boolean | —                           | false   |
-| color               | background color of the Tag          | string  | —                           | —       |
-| size                | tag size                             | string  | large / default /small      | default |
-| effect              | component theme                      | string  | dark / light / plain        | light   |
-| round               | whether Tag is rounded               | boolean | —                           | false   |
+| Name                | Description                          | Type                                                     | Default |
+| ------------------- | ------------------------------------ | -------------------------------------------------------- | ------- |
+| type                | component type                       | ^[string]`'success' \| 'info' \| 'warning' \| 'danger'`  | —       |
+| closable            | whether Tag can be removed           | ^[boolean]                                               | false   |
+| disable-transitions | whether to disable animations        | ^[boolean]                                               | false   |
+| hit                 | whether Tag has a highlighted border | ^[boolean]                                               | false   |
+| color               | background color of the Tag          | ^[string]                                                | —       |
+| size                | tag size                             | ^[string]`'large' \| 'default' \| 'small'`               | default |
```

Comment:
Here type should be `enum` , and the description needs to be synchronized to the props attribute. You can refer to the input.